python/tests/test_1_online.py

```diff
@@ -2201,6 +2201,19 @@ def test_configure_error_msgs_wrong_pw(acfactory):
     # Password is wrong so it definitely has to say something about "password"
     assert "password" in ev.data2
 
+    ac1.stop_io()
+    ac1.set_config("mail_pw", "abc")  # Wrong mail pw
+    ac1.configure()
+    while True:
+        ev = ac1._evtracker.get_matching("DC_EVENT_CONFIGURE_PROGRESS")
+        print(f"Configuration progress: {ev.data1}")
+        if ev.data1 == 0:
+            break
+    assert "password" in ev.data2
+    # Account will continue to work with the old password, so if it becomes wrong, a notification
+    # must be shown.
+    ac1.get_config("notify_about_wrong_pw") == "1"
+
```

Comment:
issue(blocking): At least on current main, this test passes also without the fix.

Though I'd say that it would be fine to not have a test for this and to only test manually once whether it's working as intended now.

Reply:
Because `assert` is forgotten before `ac1.get_config("notify_about_wrong_pw") == "1"` :) Spent some time for finding this